jedi/plugins/stdlib.py

```diff
@@ -513,6 +513,49 @@ def py__call__(self, arguments):
         )
 
 
+class PartialMethodObject(object):
+    def __init__(self, actual_value, arguments):
+        self._actual_value = actual_value
+        self._arguments = arguments
+
+    def __getattr__(self, name):
+        return getattr(self._actual_value, name)
```

Comment:
You can just remove that and use `class PartialMethodObject(ValueWrapper)` (see also for example `ClassMethodGet`).

It's really my bad that PartialObject is not a proper Wrapper, yet.